.github/workflows/build.yml

```
@@ -226,12 +222,21 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Install process user data file script dependencies
+        run: python -m pip install boto3 jinja2
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the Windows admin password
+        # pulled from AWS Parameter Store. Return the admin password to
+        # pass into Packer via command line variable.
+        run: python .github/scripts/process_user_data_file.py
       - name: Create machine image
         # This runs through the AMI creation process but does not
-        # actually create an AMI
+        # actually create an AMI.
```

Just to avoid needless deviation from the skeleton.

```suggestion
        # actually create an AMI
```

---

@mcdonnnj resolved with [64bc448](https://github.com/cisagov/windows-server-packer/pull/10/commits/64bc4486a4e52f85de2b0d4e18d5cfecf2ee06a6)